### PR TITLE
Drag race

### PR DESCRIPTION
**Pull request by johandweber**
## Description

Fortran implementation of the Sieve including README.md and
Dockerfile

## Contributing requirements

* [X] I read the contribution guidelines in CONTRIBUTING.md.
* [X] I placed my solution in the correct category folder.
* [X] I added a README.md or Dockerfile.
* [X] I selected `drag-race` as the target branch.
* [X] All code herein is licensed compatible with BSD-3.

### CODE REVIEW

File: PrimeFortran/solution_1/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+It does not have external dependencies.
+The Fortran language does not contain a bitfiled element, but allows the bitwise manipulation
+of integers. Therefore 64-bit integers are used as store for the bitfield.
+
+It is an unfaithful implementation as the sieve is not encapsulated within 
+an object.However, Fortran 2003 (or later) in principle allows for a object 
+oriented approach to the problem (I may implement it later).
+
+
+##Run instructions
```
Comment (rbergen) [inline PrimeFortran/solution_1/README.md]: Could you please add a space between the "header hashes" (\#\# in this case) and the first letter of the header text? Without the space they don't work. :)

File: PrimeFortran/solution_1/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+when using a DOS-like command line.
+
+
+For the output of the results the switch `.false.` in the call of
+`print_results` (line 60) has to be changed to `.true.`.
+(Note: In Fortran the periods before and after `true` and `false`
+are part of the boolean constants and must not be omitted.)
+
+###Dockerfile
```
Comment (rbergen) [inline PrimeFortran/solution_1/README.md]: See previous comment.

File: PrimeFortran/solution_1/README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+A Dockerfile is provided.
+
+
+
+##Output
```
Comment (rbergen) [inline PrimeFortran/solution_1/README.md]: See previous comment.

File: PrimeFortran/solution_1/README.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+are part of the boolean constants and must not be omitted.)
+
+###Dockerfile
+
+A Dockerfile is provided.
+
+
+
+##Output 
+johandweber_fortran;2739;5.002;1
```
Comment (rbergen) [inline PrimeFortran/solution_1/README.md]: Please put the output in a code block.

Comment (johandweber) [reply PrimeFortran/solution_1/README.md]: Ok, I will change README.md according to your suggestions.

**Review by rbergen — APPROVED** [review]
LGTM

**Comment by marghidanu** [comment]
LGTM #2

**Pull request merged**